config.js

```diff
@@ -9,6 +9,7 @@ var localConfig = extend(true, {
         // [Source, excluding bower_components]: [Destination].
         'document-register-element/build/document-register-element.max.js': config.LIB_DEST_PATH,
         'flipsnap/flipsnap.js': config.LIB_DEST_PATH,
+        'fxpay/dist/fxpay.min.js': config.LIB_DEST_PATH,
```

Comment:
It'll get minified whether it's a RequireJS module or not. Since we shim it below, it'll get picked up.